### ISSUE TITLE
Can't bookmark webpage

### ISSUE THREAD

**Issue by ComputerRDL** [zen-browser/desktop#3797]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

I am unable to bookmark a page. It initially seems it has been bookmarked with the pink coloring, but it doesn't appear in the bookmark's listed pages. It also doesn't give the option to choose which folder the page can be bookmarked. 

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.2 (Firefox 133.0.3) (aarch64)

### What platform are you seeing the problem on?

macOS - Intel

**Comment by djorium** [comment]
Can you provide some sort of video or something, I can't seem to reproduce this issue.

**Comment by jhtconner** [comment]
I've noticed the version of zen you're using is the arm version (aarch64) which is meant for the m-series chips as opposed to the intel version. Could be worth installing the intel mac version and seeing if that's the issue?

**Comment by ComputerRDL** [comment]
I have the M1 pro.

**Comment by machiav3lli** [comment]
Same here on:

```
Operating System: CachyOS Linux
KDE Plasma Version: 6.2.4
KDE Frameworks Version: 6.9.0
Qt Version: 6.8.1
Kernel Version: 6.12.5-1-cachyos-eevdf (64-bit)
Graphics Platform: Wayland
Processors: 8 × AMD Ryzen 7 4700U with Radeon Graphics
Memory: 15.0 GiB of RAM
Graphics Processor: AMD Radeon Graphics
```

**Comment by iuvi7** [comment]
Same problem for me too

**Comment by mr-cheffy** [comment]
did you make sure you didnt select a different workspace for the bookmark? Again, a video would help to better visualize the issue

**Comment by iuvi7** [comment]
> did you make sure you didnt select a different workspace for the bookmark? Again, a video would help to better visualize the issue

I am bad with computers so briefly:

I updated Zen to 1.0.2-b.2 (2024-12-14) 64-bit Windows version **without changing any settings by me**

Then, i notice:

If i use Ctrl+D or just push on Star in address bar - it get blue, but it **does not** actually add bookmark to my vault anymore. Also i can not choose to wich folder to save it anymore, no more fallout window to do so. In previous version\release it was works just fine

**Comment by fidodido48** [comment]
Same here.
1. 0 bookmarks being imported from other browser (despite having plenty in chrome).
2. Nothing happens when trying to restore from backup (via JSON file). No error, but also no bookmarks showing up. Silent fail it seems.
3. "_Unable to proces file_" error when trying to import via HTML. 

OS: Arch Linux, KERNEL: 6.12.3-zen1-1-zen
zen-browser-bin: 1.0.2.b.5-1 (from AUR)

**Comment by iuvi7** [comment]
It start's working for me, **but only** when i create _absolutly new profile_, after wich i was need to transfer all my bookmarks, tabs, themes and addons - **taked a lot of time** :(

**Comment by fidodido48** [comment]
> It start's working for me, **but only** when i create _absolutly new profile_, after wich i was need to transfer all my bookmarks, tabs, themes and addons - **taked a lot of time** :(

Yep. Creating new profile indeed helped (but that's just a workaround). Thx.
